### ISSUE TITLE
todo: implement lightweight full text/fuzzy search

### ISSUE THREAD

**Issue by sw-yx** [sw-yx/swyxkit#45]
both typesense and algolia require api key signup, which is bleh, so let's try to handroll it like chumps: https://github.com/sw-yx/netlify-plugin-search-index

**Comment by git-no** [comment]
How would be this [fuse.js](https://fusejs.io)? It works fine in my app and is super fast.

**Comment by sw-yx** [comment]
yeah i use fusejs under the hood in that thing - welcome to help implement it!

**Comment by VaithiSniper** [comment]
so if i'm reading this right, we can rework the search with fuse.js?

**Comment by VaithiSniper** [comment]
can i take it up if so?

**Comment by VaithiSniper** [comment]
#122 added fuse.js
and is this what's expected?

**Comment by sw-yx** [comment]
yup!

**Comment by sw-yx** [comment]
see also the ideas and comments in https://github.com/sw-yx/swyxkit/issues/80
